packages/twenty-server/src/engine/metadata-modules/remote-server/utils/validate-remote-server-input.ts

```diff
@@ -13,7 +13,7 @@ export const validateObject = (input: object) => {
   }
 };
 
-export const validateString = (input: string) => {
+export const validateStringRemoteServerInput = (input: string) => {
```

Comment:
I renamed this but I am not sure what this validation is for on the id? @thomtrp